pennylane_lightning/lightning_tensor/_measurements.py

```diff
@@ -95,7 +114,9 @@ def var(self, measurementprocess: MeasurementProcess):
             Variance of the observable
         """
         if isinstance(measurementprocess.obs, qml.SparseHamiltonian):
-            raise NotImplementedError("Sparse Hamiltonian Observables are not supported.")
+            raise NotImplementedError(
```

Comment:
Where is test for this exception added?

Reply:
yea, it is tested [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/1c69f8fc3c9eafbb669b594f8afe961e0df1a43b/tests/lightning_tensor/test_gates_and_expval.py#L259-L263)